src/plugins/intel_cpu/src/utils/cpu_utils.hpp

```diff
@@ -47,6 +49,18 @@ inline std::vector<size_t> getNormalizedDimsBySize(const InferenceEngine::SizeVe
     return normalizedDims;
 }
 
+/**
+* @brief Returns model type
+* @param model
+* analysed model
+* @return model type
+*/
+inline Config::ModelType getModelType(const std::shared_ptr<const Model>& model) {
```

Comment:
There is no point to have it in cpu_utils.hpp
We should just create a static function in plugin.cpp

Reply:
fixed